plugin.rb

```diff
@@ -14,7 +16,11 @@
 enabled_site_setting :salesforce_enabled
 
 register_asset 'stylesheets/salesforce.scss'
-register_svg_icon "fab-salesforce" if respond_to?(:register_svg_icon)
+
+if respond_to?(:register_svg_icon)
```

Comment:
I don't think we need this check - all versions of discourse now support `register_svg_icon`